hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -919,6 +919,11 @@ private Response createMultipartKey(OzoneVolume volume, String bucket,
         throw newError(NO_SUCH_UPLOAD, uploadID, ex);
       } else if (isAccessDenied(ex)) {
         throw newError(S3ErrorTable.ACCESS_DENIED, bucket + "/" + key, ex);
+      } else if ((ex.getResult() == ResultCodes.INVALID_PART)) {
+        OS3Exception os3Exception = S3ErrorTable.newError(
```

Comment:
Thanks for the patch @xichen01. 
nit: Use statically imported method `newError` instead of `S3ErrorTable.newError` here to make the code more consistent.

Could you also add Unit Tests for uploading Partitions with invalid partition id?